tests/smoke/examples/test_notebooks_pyspark.py

```diff
@@ -59,4 +59,4 @@ def test_mmlspark_lightgbm_criteo_smoke(notebooks, output_notebook, kernel_name)
     results = sb.read_notebook(output_notebook).scraps.dataframe.set_index("name")[
         "data"
     ]
-    assert results["auc"] == pytest.approx(0.68895, rel=TOL, abs=ABS_TOL)
+    assert results["auc"] == pytest.approx(0.65, rel=TOL, abs=ABS_TOL)
```

Comment:
this error is weird, we have been running this notebook for a couple of years, maybe there are some default changes in the new version of LightGBM? I ping Mark and Ilia in the issue, let's see if they reply before making the change